ptx/sec_vector_intro.ptx

```diff
@@ -420,7 +420,7 @@
 
                 <li>
                   <p>
-                    The scalar product of <m>c</m> and <m>\vec v</m> is the vector
+                    The scalar multiplication of <m>c</m> and <m>\vec v</m> is the vector
```

Comment:
Just a comment as long as I am reading; feel free to ignore.

This phrasing now sounds awkward. How about just `The product of <m>c</m> and <m>\vec v</m>...`? If you change, also a few lines down.

Reply:
I think the intent from @APEXCalculus is to introduce the term "scalar multiplication" here. Maybe there's a better way to do it? Something like, "the product of the vector v with the scalar c is known as scalar multiplication"?

Reply:
I've given new wording in PR #253 (it took me 2 commits to get it right). It says "The multiplication of a scalar <m>c</m> and a vector <m>\vec v</m> is the vector ... ". The sentence below that definition uses the term 'scalar multiplication', which is implied by the definition.